### ISSUE TITLE
Google login returns undefined

### ISSUE THREAD

**Issue by joelfi** [cyraxx/pogobuf#119]
Logging in with GoogleLogin returns `undefined`, even if the credentials are correct. No errors given. This bug is occurring only in version 1.9.0 which suggests we should look into [differences between 1.9.0 and 1.8.0](https://github.com/cyraxx/pogobuf/compare/98d1e1672b99e45b2889187bc507821898f472a8...2e17c012231d0eb8d0de746a4d0c4b3618d4c575).

Test code:
```js
new require('pogobuf').GoogleLogin().login('username@gmail.com', 'password').then(console.log).catch(console.error);
```

Result in 1.9.0: `undefined`
Result in 1.8.0: `[google token]`

**Comment by joelfi** [comment]
Fixed with 11057d5f18ad9af59a48b8a08d96cdad049dd964

**Comment by cyraxx** [comment]
Just published 1.9.1 fixing this.
